packages/web-react/src/components/UNSTABLE_Truncate/README.md

```diff
@@ -3,34 +3,110 @@
 > ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
 > Please use it with caution.
 
-Truncate is a component that truncates text based on defined number of rows.
+Truncate is a component that truncates text based on different criteria: lines, words, or characters.
 
 ```jsx
 import { UNSTABLE_Truncate } from '@lmc-eu/spirit-web-react';
 
 <UNSTABLE_Truncate>{/* Text go here */}</UNSTABLE_Truncate>;
 ```
 
-## Lines
+## Truncation Modes
 
-You can add the number of lines to which you want to truncate the text using `lines` props
+The component supports three different truncation modes using the `mode` prop:
+
+### Lines
+
+Truncate text to a specific number of lines using `mode="lines"` and `limit` props:
+
+```jsx
+<UNSTABLE_Truncate limit={2} mode="lines">
+  {/* Text go here */}
+</UNSTABLE_Truncate>
+```
+
+### Words
+
+Truncate text to a specific number of words using `mode="words"` and `limit` props:
 
 ```jsx
+<UNSTABLE_Truncate limit={10} mode="words">
+  {/* Text go here */}
+</UNSTABLE_Truncate>
+```
+
+ℹ️ Please note that the word and character truncation modes work only when the children are strings (not React elements).
```

Comment:
I think this is important and deserves more details. Please add a few examples (or maybe a table overview) of what will and will NOT work, like text with emphasis, links, lists, icons, and when.